.changeset/new-doors-fold.md

```diff
@@ -0,0 +1,5 @@
+---
+"@spectrum-css/combobox": minor
+---
+
+Combobox now includes a `read-only` state. Add the `is-readOnly` class to the Combobox to enable this feature.
```

Comment:
It looks like the `readonly` attribute is also needed on the input to use the feature, and we should make it clear that it's more than adding the class.